Jenkinsfile

```diff
@@ -46,17 +46,17 @@ pipeline {
             }
         }
 
-        stage('Freighter Tests') {
-            steps {
-                timeout(60) {
-                    sh '''
-                        export ARTIFACTORY_USERNAME=\"\${ARTIFACTORY_CREDENTIALS_USR}\"
-                        export ARTIFACTORY_PASSWORD=\"\${ARTIFACTORY_CREDENTIALS_PSW}\"
-                        ./gradlew freighterTest --info --stacktrace --no-daemon
-                        '''
-                }
-            }
-        }
+//         stage('Freighter Tests') {
```

Comment:
just passing through as i've been asked to do some release support on this repo and was looking at this PR ...

These are commented out why ??

I know we have had issues with Freighter test stability in other places recently but are they a supported set of tests in this repo or not, If the answer is no should these just be removed as having them commented out just causes confusion for the next person that comes along? 

If they are support and should be working is there any plan to address this? If we are at a stage of accepting commenting them out to get PRs/ release to pass are they providing any value here?

Again i know relatively little about Freighter tests so maybe all this has been answer in conversations im not aware of!

Maybe @adelel1 might have some also

Reply:
My most recent commit on target branch adds a parameter to turn these on or off and are currently off by default due to stability issues, so you will pick that up when rebase and accept incoming changes

Reply:
Thanks, I don't really have any more context than you. I agree, I would leave them in uncommented and switched off by default. Until they are eventually enabled. Not aware of them not being supported. No current plans to investigate them though.